opentasks/src/main/java/org/dmfs/tasks/widget/TextFieldView.java

```diff
@@ -106,4 +106,9 @@ public void onContentChanged(ContentSet contentSet)
 			}
 		}
 	}
+
+	public String getText()
+	{
+		return mText.getText().toString();
```

Comment:
Ok, I see, that felt a little bit wrong to me too, but on the other hand I thought user would probably want to search for location that is displayed, whatever that is.
But I changed `TextFieldView` now to save the actual text in a field and use that for the getter. Please check if it's good like that.